pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -339,7 +339,10 @@ class OSRow extends React.Component {
             return ({
                 toString: function() { return this.displayName },
                 compareTo: function(value) {
-                    return value.shortId.toLowerCase().includes(this.shortId) || value.displayName.toLowerCase().includes(this.displayName);
+                    if (typeof value == "string")
+                        return this.shortId.toLowerCase().includes(value.toLowerCase()) || this.displayName.toLowerCase().includes(value.toLowerCase());
+                    else
+                        return this.shortId.toLowerCase().includes(value.shortId.toLowerCase()) || this.displayName.toLowerCase().includes(value.displayName.toLowerCase());
```

Comment:
This reverses the "includes" direction, was that really intended?

Reply:
Oh it's partially correct: 
value can be the text search string or another option. In the latter case we need exact match, not includes.

Reply:
I will repost.

Reply:
So I reposted and fixed the second check. I will probably adjust it in PF4 again, I am not satisfied with how this component is implemented. But at least right now it works as expected.